iap/validate_jwt.py

```diff
@@ -27,18 +27,41 @@
 import requests
 
 
-def validate_iap_jwt(base_url, iap_jwt):
+def validate_iap_jwt(cloud_project_number, iap_jwt,
+                     app_engine_project_id=None,
+                     compute_engine_backend_service_id=None):
     """Validate a JWT passed to your application by Identity-Aware Proxy.
 
+    One and only of of (app_engine_project_id, compute_engine_backend_service_id)
+    must be set.
+
     Args:
-      base_url: The URL from the incoming request, minus any path, query, etc.
-                For instance: "https://example.com:8443" or
-                "https://example.appspot.com" .
-      iap_jwt: The contents of the X-Goog-Authenticated-User-JWT header.
+      cloud_project_number: The project *number* for your Google Cloud project.
```

Comment:
Oh no, project number? That's awful! Why is that needed instead of just the project ID?